agent/build_session_test.go

```diff
@@ -142,7 +142,7 @@ func TestCleandirCommand(t *testing.T) {
 	}
 
 	for i, f := range matches {
-		actual := f[len(wd)+1:]
+		actual := f[len(wd) + 1:]
```

Comment:
Could you use gofmt to format code? So that we won't have these tiny changes when we save file.
gofmt is standard format in Go.

Sent from my iPhone

> On May 27, 2016, at 6:40 PM, ylzheng <notifications@github.com> wrote:
> 
> In agent/build_session_test.go:
> 
> > @@ -142,7 +142,7 @@ func TestCleandirCommand(t *testing.T) {
> >  	}
> >  
> >  	for i, f := range matches {
> > -		actual := f[len(wd)+1:]
> > +		actual := f[len(wd) + 1:]
> This is the IDEA auto format result.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>